doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,194 @@
+   * ``pri-dir-gpios`` - GPIO characteristic of the device that controls the ``PRIORITY`` signal of the PTA (priority + RX/TX direction).
+   * ``grant-gpios`` - GPIO characteristic of the device that controls the ``GRANT`` signal of the PTA (RF medium access granted).
+
+   Type ``phandle-array`` is used here, which is common in Zephyr's devicetree to describe GPIO signals.
+   The first element ``&gpio0`` refers to the GPIO port ("port 0" has been selected in the example shown).
+   The second element is the pin number on that port.
+
+#. On nRF53 devices, you must also apply the same devicetree node mentioned in step 2 to the network core.
+   To do so, apply the overlay to the correct network core child image by creating an overlay file named :file:`child_image/*childImageName*.overlay` in your application directory, for example :file:`child_image/multiprotocol_rpmsg.overlay`.
+
+   The ``*childImageName*`` string must be one of the following values:
+
+   *  ``multiprotocol_rpmsg`` for multiprotocol applications having support for both 802.15.4 and Bluetooth.
+   *  ``802154_rpmsg`` for applications having support for 802.15.4, but not for Bluetooth.
+   *  ``hci_rpmsg`` for application having support for Bluetooth, but not for 802.15.4.
+
+
+
+WiFi Coexistence implementations
+********************************
+
+The following CX implementations are available:
+
+* :ref:`Generic 3-pin interface`
+* :ref:`"Thread" implementaion`
+
+Each implementaion is a plugin consisting of one or more C source files interfacing with the MPSL API (which in turn communicates with the
+802.15.4 Radio Driver)
+
+When either CX variant is enabled, the driver requests access to the RF medium from the PTA and informs it about the operation it's about
+to perform. It also reacts properly to the information from the PTA that access to RF medium was granted/denied.
+
+Generic 3-pin interface
+=======================
+
+This implementation uses the following pins to communicate with the PTA:
+
+* REQUEST pin - output controlled by the protocol driver, it is asserted in order to request RF access from the PTA. It can be configured by
+  setting the `req-gpios` property of the `nrf_radio_coex` device tree node.
+* PRIORITY pin - output controlled by the protocol driver, multiplexes the information about the operation priority and the type of operation
+  (RX/TX) to perform. It can be configured by setting the `pri-dir-gpios` property of the `nrf_radio_coex` device tree node. Currently only
+  priority "low" is supported.
+* GRANT pin - output controlled by the PTA, asserts, when PTA grants access to RF to the 802.15.4 and deasserts when it denies the access. It can
+  be configured by setting the `grant-gpios` property of the `nrf_radio_coex` device tree node.
+
+The support for this interface is provided in a single-file plugin located in the NRF Connect SDK nrf repo, in the subsys/mpsl/cx directory.
```

Comment:
Why not use `sdk-nrf` name instead of `NRF Connect SDK nrf`? It's unambiguous then

Reply:
I wanted to use a more "official" name, or try to emphasize it was in NCS, however there is probably no sense to do that, so changed as you suggested.

Reply:
Yeah, I get it. The problem is though that "nRF Connect SDK" is rather a conglomerate of repositories than a single repo, so if you want to refer to a repository, I think it's better to use its precise name. Thanks for changing it :) there's one more such occurrence below.